src/Menu.js

```diff
@@ -1,126 +1,136 @@
 import React, {Component} from 'react';
 import { 
-  Alert, Text, SafeAreaView, SectionList, Linking, AsyncStorage
+  Alert, SectionList, Linking, AsyncStorage
 } from 'react-native';
 import ListItem from './components/listitem';
+import {ThemeText,ThemeBackground} from './components/components';
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [A space is required after ','. (comma-spacing)](https://app.codacy.com/app/s-owl/skhu-app/pullRequest?prid=4440877)